Library/Homebrew/cmd/test-bot.rb

```diff
@@ -244,22 +241,9 @@ def safe_formulary(formula)
     end
 
     def git(*args)
-      rd, wr = IO.pipe
-
-      pid = fork do
-        rd.close
-        STDERR.reopen("/dev/null")
-        STDOUT.reopen(wr)
-        wr.close
-        Dir.chdir @repository
-        exec("git", *args)
+      Dir.chdir(@repository) do
+        Utils.popen_read("git", *args)
       end
```

Comment:
I wonder if we should use `git -C @repository args...` instead. From the manpage:
```
-C <path>
       Run as if git was started in <path> instead of the current working directory.
```